expected/wasm32-wasi/defined-symbols.txt

```diff
@@ -276,6 +276,7 @@ _exit
 _flushlbf
 _initialize
 _start
+_start
```

Comment:
Why does `_start` appear twice now?

Reply:
It's defined in crt1.o and crt1-reactor.o. Programs will only use one or the, but the script which generates defined-symbols.txt doesn't know that.